src/main/java/com/marmelab/quoridor/web/QuoridorEndpoint.java

```diff
@@ -0,0 +1,27 @@
+package com.marmelab.quoridor.web;
+
+import com.marmelab.quoridor.game.Game;
+import com.marmelab.quoridor.model.Board;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("quoridor")
+public class QuoridorEndpoint {
+
+    private Game game1;
+
+    @GetMapping
+    public String getGame(Model model) {
+        if (game1 == null) {
```

Comment:
Why do you sometimes use this to refer to own class attributes and sometimes not?